### PR TITLE
Enable saving reconstruction outputs as TIFF

### PR DESCRIPTION
**Pull request by thomasmfish**
DV file is a bit specialised and it was fairly easy to make this change using the existing code. Had some trouble with the channels being saved as separate images, according to the OME-TIFF metadata, but that's fixed now.

### CODE REVIEW

File: src/sim_recon/images/tiff.py [hunk 1]
```diff
@@ -101,25 +117,26 @@ def get_channel_dict(channel: ImageChannel) -> dict[str, Any] | None:
...
+            1e4 / resolution.x,
+            1e4 / resolution.y,
         )
         tiff_kwargs["resolutionunit"] = (
             tf.RESUNIT.CENTIMETER
         )  # Use CENTIMETER for maximum compatibility
 
     if ome:
-        if xy_pixel_size_microns is not None:
+        tiff_kwargs["metadata"]["Name"] = output_path.name
```
Comment (PeterC-DLS) [inline src/sim_recon/images/tiff.py]: Stylistically, I would define `tiff_metadata = tiff_kwargs["metadata"]` as Python is interpreted

File: src/sim_recon/recon.py [hunk 2]
```diff
@@ -207,7 +210,22 @@ def _reconstructions_to_output(
...
+        suffix = ".dv"
+        write_output = partial(
+            write_dv,
+            input_dv=input_dv,
+            overwrite=overwrite,
+        )
+    elif file_type == "tiff":
+        suffix = ".ome.tiff"
+        write_output = partial(write_tiff, ome=True, overwrite=overwrite)
+        del input_dv
```
Comment (PeterC-DLS) [inline src/sim_recon/recon.py:227]: If not needed then only read in first clause

File: src/sim_recon/images/tiff.py (hunk 1 above)
Comment (thomasmfish) [reply src/sim_recon/images/tiff.py]: Okay, I've changed it to avoid the double indexing

**Review by PeterC-DLS — APPROVED** [review]
LGTM

**Pull request merged**